swagger.yaml

```diff
@@ -26,6 +26,19 @@ paths:
       responses:
         201:
           description: "teacher added"
+  /student:
+    post:
```

Comment:
It's ok... but try to keep student and teacher routws defined separately

Reply:
Okk on it